Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -384,11 +417,11 @@ protected virtual void OnHealthChanged(DamageChangedEventArgs e)
             Dirty();
         }
 
-        void IRadiationAct.RadiationAct(float frameTime, SharedRadiationPulseComponent radiation)
+        public void RadiationAct(float frameTime, SharedRadiationPulseComponent radiation)
         {
             var totalDamage = Math.Max((int)(frameTime * radiation.RadsPerSecond), 1);
 
-            ChangeDamage(DamageType.Radiation, totalDamage, false, radiation.Owner);
+            ChangeDamage(GetDamageType("Radiation"), totalDamage, false, radiation.Owner);
```

Comment:
The damage type to deal should be a DataField (that defaults to "Radiation" probably).
I'm sure there are better solutions though.